tests/parm/nems.configure.cpld.IN

```diff
@@ -74,6 +74,7 @@ WAV_attributes::
   logfile = wav.log
   mesh_wav = @[MESH_WAV]
   multigrid = @[MULTIGRID]
+  user_sets_restname = true
```

Comment:
@DeniseWorthen It's the goal from the WW3 side for the restart names to be controlled by a WW3 input file instead of a "nems.configure" restart file.   Just wanted to make you aware that is a planned change from the WW3 side in the  near future.

Reply:
It is good to know the goal on WW3 side. I think we can get this change in at this time. We can review it when the ww3 restart filename changes are ready. the purpose here is to have consistent restart file names as much as possible for all the model components. Thanks.